python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -2091,20 +2089,19 @@ def __setattr__(self, attr, value):
 
             # reduce duplicate values here, leave only duplicates for LCG,
             # where we can have replicas
-            uniqueValuesDict = []
-            uniqueValues = []
+            uniqueValuesDict = oDict()
```

Comment:
Rather than making the code use an ordered dict (when outputfiles aren't inherently ordered), perhaps just fix the `testNoFilesSpecifiedAllSame` test to compare sorted output?